src/components/Cards/VideoCard.jsx

```diff
@@ -14,16 +14,16 @@ const VideoCard = ({ video }) => {
           style={{
             opacity: mouseOnCard === true ? "50%" : "100%",
```

Comment:
UI suggestion having opacity 50% feels like the row is disabled so maybe don't change the opacity of the Text inside of it.